src/rdbms/setup.py

```diff
@@ -8,7 +8,7 @@
 from codecs import open
 from setuptools import setup, find_packages
 
-VERSION = "0.0.4"
+VERSION = "0.0.5"
 
```

Comment:
Change the description and long_description to "Support for Azure MySQL and Azure PostgreSQL." (or something else) to conform to the new guidelines on this - https://github.com/Azure/azure-cli-extensions/blob/master/docs/extension_summary_guidelines.md

Reply:
Also, mark the extension as preview which I think this is.
https://github.com/Azure/azure-cli/blob/dev/doc/extensions/metadata.md#azextispreview